common/views/components/SearchBar/new.tsx

```diff
@@ -12,7 +12,6 @@ import Typed from 'typed.js';
 import { search } from '@weco/common/icons';
```

Comment:
np but could we rename this file to `Searchbar.New.tsx`? Potentially could use the variant model we've started having in components likes [Tabs](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/views/components/Tabs/index.tsx) or [CopyButtons](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/views/components/CopyButtons/index.tsx)